server/app.js

```diff
@@ -7,7 +7,8 @@ require("dotenv").config();
 var cors = require("cors");
 const { insertSampleData, deleteAllDocuments } = require("./bin/defaultdata");
 
-const mongoDB = process.env.MONGODB_CONN_STRING;
+// const mongoDB = process.env.MONGODB_CONN_STRING;  
+const mongoDB = "mongodb://localhost:27017/assoceasedb";  
```

Comment:
This should probably be changed back to the "correct" way.

Reply:
True true, I'm modifying this